frontend/packages/kubevirt-plugin/integration-tests/v2v/vmware/v2v.vmware.basic.import.scenario.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  flavorWorkloadConfigs,
+  vmwareSecondVMConfig,
+  vmwareVMConfig,
+  vmwareVMMultiNicConfig,
+} from './v2v.configs';
+import {
+  withResource,
+  createResources,
+  deleteResources,
+} from '@console/shared/src/test-utils/utils';
+import { VmwareImportWizard } from '../../tests/models/vmwareImportWizard';
+import { multusNAD } from '../../tests/mocks/mocks';
+import { VM_STATUS } from '../../tests/utils/constants/vm';
+import { V2V_VM_IMPORT_TIMEOUT } from '../../tests/utils/constants/common';
+
+describe('Kubevirt migrate VM from VMWare using wizard', () => {
+  const leakedResources = new Set<string>();
+  const wizard = new VmwareImportWizard();
+
+  beforeAll(async () => {
+    createResources([multusNAD]);
+  });
+
+  afterAll(async () => {
+    deleteResources([multusNAD]);
+  });
+
+  it(
+    'Imports VM from VMware Instance, basic test',
+    async () => {
+      const vm = await wizard.import(vmwareVMConfig);
+      await withResource(leakedResources, vm.asResource(), async () => {
```

Comment:
Never seen this before, I will try to re-run again

Reply:
I am still seeing this on Igor's env, and since he is not seeing it - there might be something wrong with my setup. Since other things are working fine, we should not block this PR on this and resolve this one once it can be reproduced.